src/vivarium_gates_nutrition_optimization/components/pregnancy.py

```diff
@@ -0,0 +1,21 @@
+from vivarium_public_health.disease import DiseaseModel, DiseaseState, SusceptibleState
+
+from vivarium_gates_nutrition_optimization.constants import data_keys, models
+
+
+def Pregnancy():
+    not_pregnant = SusceptibleState(models.PREGNANCY_STATE)
```

Comment:
Keep in mind that the `not_pregnant.name()` is probably `susceptible_to_pregnant` not `not_pregnant`